manager/integration/tests/test_rwx.py

```diff
@@ -1,12 +1,373 @@
+                               }
+                            }
+                         }
+                      ],
+                      "imagePullPolicy": "IfNotPresent",
+                      "readinessProbe": {
+                         "exec": {
+                            "command": [
+                               "ls",
+                               "/export"
+                            ]
+                         },
+                         "initialDelaySeconds": 5,
+                         "periodSeconds": 5
+                      },
+                      "livenessProbe": {
+                         "exec": {
+                            "command": [
+                               "ls",
+                               "/export"
+                            ]
+                         },
+                         "initialDelaySeconds": 5,
+                         "periodSeconds": 5
+                      },
+                      "volumeMounts": [
+                         {
+                            "name": "export-volume",
+                            "mountPath": "/export"
+                         }
+                      ]
+                   }
+                ],
+                "volumes": [
+                   {
+                      "name": "export-volume",
+                      "persistentVolumeClaim": {
+                         "claimName": "longhorn-nfs-provisioner"
+                      }
+                   }
+                ],
+                "terminationGracePeriodSeconds": 30,
+                "tolerations": [
+                   {
+                      "effect": "NoExecute",
+                      "key": "node.kubernetes.io/not-ready",
+                      "operator": "Exists",
+                      "tolerationSeconds": 60
+                   },
+                   {
+                      "effect": "NoExecute",
+                      "key": "node.kubernetes.io/unreachable",
+                      "operator": "Exists",
+                      "tolerationSeconds": 60
+                   }
+                ]
+             }
+          }
+       }
+    }
+
+    def finalizer():
+        api = get_apps_api_client()
+        delete_and_wait_deployment(api, 'longhorn-nfs-provisioner',
+                                   namespace='longhorn-system')
+
+    request.addfinalizer(finalizer)
+
+    return deployment_nfs_manifest
+
+
+@pytest.fixture(scope="module", autouse="True")
+def nfs_pvc(request):
+    pvc_manifest = {
+        'apiVersion': 'v1',
+        'kind': 'PersistentVolumeClaim',
+        'metadata': {
+            'name': 'longhorn-nfs-provisioner',
+            'namespace': 'longhorn-system'
+        },
+        'spec': {
+            'storageClassName': 'longhorn',
+            'accessModes': [
+                'ReadWriteOnce'
+            ],
+            'resources': {
+                'requests': {
+                    'storage': '20G'
```

Comment:
For the test environment you can use a tiny storage amount, for the longhorn volume.